apps/www/components/blog/blog-code-block.tsx

```diff
@@ -63,54 +63,21 @@ export function BlogCodeBlock({ className, children }: any) {
         {blocks.map((block: any, index: number) => {
           return (
             <TabsContent value={buttonLabels[index]} key={buttonLabels[index]} className="pr-12">
-              <Highlight
-                theme={darkTheme}
-                code={block.children}
+              <SyntaxHighlighter
                 language={block.className.replace(/language-/, "")}
+                style={darkTheme}
+                showLineNumbers={true}
+                wrapLongLines={true}
+                wrapLines={true}
+                lineProps={(lineNumber) => ({
```

Comment:
remove this we don't need the on click stuff

Reply:
Removed